tests/classes/export/csv_test.php

```diff
@@ -148,41 +148,44 @@ public function test_one_stage(): void {
      * Two stages with final agreed grade, extension not enabled
      */
     public function test_two_stages(): void {
-
+        $timenow = time();
```

Comment:
https://moodledev.io/docs/4.4/apis/core/clock

Reply:
@watson8 log a follow up story for this. I will merge as is.

Reply:
@aspark21 story open CTP-3853